tests/test_commands.py

```diff
@@ -151,17 +151,18 @@ def test_pull_with_branch_no_git_repo(self, project_mock, log_mock, bmock):
     @patch('txclib.utils.get_current_branch')
     @patch('txclib.commands.logger')
     @patch('txclib.commands.project.Project')
-    def test_pull_only_branchname_option(self, project_mock, log_mock, bmock):
-        bmock.return_value = None
+    def test_pull_branch_git_repo(self, project_mock, log_mock, bmock):
+        bmock.return_value = 'a-branch'
         pr_instance = MagicMock()
         project_mock.return_value = pr_instance
-        with self.assertRaises(SystemExit):
-            cmd_pull(['--branchname', 'somebranch'], '.')
-        log_mock.error.assert_called_once_with(
-            "--branchname options should be used along with "
-            "the --branch option."
+        cmd_pull(['--branch'], '.')
+        assert pr_instance.pull.call_count == 1
```

Comment:
I believe we should avoid plain assertions and use the unittest's ones

Reply:
I googled this but I couldn't find any strong reason not to use it. Do you have a specific reason why not to?

Reply:
+1

Reply:
@SofiaMargariti we are already using unittest's assertions on the same file, so why not be consistent?

Reply:
@SofiaMargariti for consistency most importantly, but there is also a real difference as mentioned [here](https://stackoverflow.com/a/2958183)